docs/adr/0061-cozy-integration-set-up.md

```diff
@@ -57,16 +57,45 @@ cozy-stack instances add tmail.localhost:8080 --passphrase cozy --apps home,stor
 1. On the cozy-twakemail side
 - Clone `https://github.com/cozy/cozy-twakemail`
 - `yarn install`
-- In `src/components/AppLayout.jsx`, replace `flag('mail.embedded-app-url')` with `http://localhost:2023`
 - `yarn build`
 - `cozy-stack serve --appdir tmail:build/ --disable-csp`
+- Open another terminal while cozy-stack is running
+- `cozy-stack apps install dataproxy --domain tmail.localhost:8080`
+- `cozy-stack feature flags --domain tmail.localhost:8080 '{"cozy.search.enabled": true}'`
+- `cozy-stack feature flags --domain tmail.localhost:8080 '{"mail.embedded-app-url": "http://localhost:2023"}'`
 
 2. On the tmail side
-- Config tmail to run on basic auth
+- Config tmail to run on basic auth (possible on dev environment)
```

Comment:
adding patch to help precise configuration for basic auth and dont forget adding guide for chaging session URL in jmap-dart-client